fvm/state/accounts.go

```diff
@@ -233,13 +234,20 @@ func contractKey(contractName string) string {
 }
 
 func (a *Accounts) getContract(contractName string, address flow.Address) ([]byte, error) {
+
 	contract, err := a.getValue(address,
 		true,
 		contractKey(contractName))
 	if err != nil {
 		return nil, newLedgerGetError(contractName, address, err)
 	}
 
+	log.Info().
+		Str("address", address.String()).
+		Str("contract", contractName).
+		Int("contract_len", len(contract)).
+		Msg(fmt.Sprintf("TEMP LOGGING: a contract returned for %s.%s", address.String(), contractName))
```

Comment:
Nice. I'm not sure if the flow-go codebase usually uses the global/default logger, maybe inject it into Accounts instead. Other contributors to flow-go might now better

Reply:
thats true, but not sure if we want to go through that rabbit hole for this patch to v0.13, I have it on my list for the fvm to have a proper logger and injected to components.

Reply:
I'll create another PR for the master branch today.